content/readme.md

```diff
@@ -12,9 +12,9 @@ Each day is a 1.5 to 2 hours hands-on introduction
 to MNE-Python.
 
 Here's a brief break-down of what's coming up:
-* Day 1: [Raw to evoked](raw_to_evoked/readme)
-* Day 2: [Preprocessing](preprocessing/readme)
-* Day 3: [Source Modeling](evoked_to_stc/readme)
+* Day 1: [Raw to evoked](raw_to_evoked/readme.md)
+* Day 2: [Preprocessing](preprocessing/readme.md)
+* Day 3: [Source Modeling](evoked_to_stc/readme.md)
```

Comment:
I thought nobody would notice :p Apparently not

We can't do this though :-( It would break the building of the jekyll site as it expects no `.md`. Let me push a change so that the README is modified during the copy to master.